src/apps/demo-app/package.json

```diff
@@ -17,7 +17,7 @@
     "@angular/common": "file:../../../node_modules/@angular/common",
     "@angular/compiler": "file:../../../node_modules/@angular/compiler",
     "@angular/core": "file:../../../node_modules/@angular/core",
-    "@angular/flex-layout": "file:../../../dist/releases/flex-layout",
+    "@angular/flex-layout": "file:../../../dist/releases/flex-layout/angular-flex-layout.tgz",
```

Comment:
Why is a `.tgz` needed here ?

Reply:
To maintain parity with the Universal task, you'll see it has the same logic. There was some reason in the Universal app why direct directory access wasn't working (I think a mismatch in Angular versioning/linking issue), so installing it was the only way to get it to function correctly.